src/components/cart_product_card/CartProductCard.jsx

```diff
@@ -5,20 +5,42 @@ import {
 	changeQuantity,
 } from "../../api/apicall";
 import { useProductContext } from "../../context/ProductContext";
+import { Toast } from "../toast/Toast";
```

Comment:
Suggestion, you can export components and context files from index.js and throughout your codebase only import from index.js

Reply:
Implemented suggestion in new commit.